### PR TITLE
Add React to the Get Set Up Guides

### PR DESCRIPTION
**Pull request by robertnicolo-okta**
## Description:

Add React to the Get Set Up Guides

### Resolves:

* [OKTA-515730](https://oktainc.atlassian.net/browse/OKTA-515730)

### Deploy link (09/19/2022)

**Get Set Up:** https://6328b414a2eb0e0090784286--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-org-setup/react/main/

**Download and set up the SDK ..:** https://6328b414a2eb0e0090784286--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-download-setup-app/react/main/

**Run the sample apps:** https://6328b414a2eb0e0090784286--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-run-samples/react/main/

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/githubinstructions.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+```shell
+git clone https://github.com/okta/okta-auth-js.git
+```
+
+Important files, directories, and projects located in the `okta/okta-auth-js` repository include:
+
+* [`/docs/idx.md`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md): Identity Engine introduction and reference
+* [`/samples/generated`](https://github.com/okta/okta-auth-js/tree/master/samples/generated): Directory of sample apps
+* [`/samples/generated/express-embedded-auth-with-sdk`](https://github.com/okta/okta-auth-js/tree/master/samples/generated/express-embedded-auth-with-sdk): Express sample app using the Embedded SDK
+* [`/samples/generated/express-embedded-sign-in-widget`](https://github.com/okta/okta-auth-js/tree/master/samples/generated/express-embedded-sign-in-widget): Express sample app using the Embedded Okta Sign-in Widget
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/githubinstructions.md:12]:
```suggestion
* [`/samples/generated/express-embedded-sign-in-widget`](https://github.com/okta/okta-auth-js/tree/master/samples/generated/express-embedded-sign-in-widget): Express sample app using the Embedded Okta Sign-In Widget
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# yarn
+yarn add @okta/okta-auth-js
+
+# npm
+npm install --save @okta/okta-auth-js
+```
+
+#### 2: Set the configuration settings
+
+Set the SDK's configurations values described in [Set the configuration values](#set-the-configuration-values).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md:20]:
```suggestion
Set the SDK's configuration values described in [Set the configuration values](#set-the-configuration-values).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+```javascript
+const oktaAuth = new OktaAuth({
+  // config
+})
+```
+
+#### 4: Review the SDK methods
+
+Start integrating the available authentication workflows by reviewing the Auth JS SDK methods. See [Usage](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#usage) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md:32]:
```suggestion
Start integrating the available authentication workflows by reviewing the Auth JS SDK methods. See [Usage](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#usage).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/widgetforyourapp.md [hunk 4]
```diff
@@ -0,0 +1 @@
+See [Install the SDK](/docs/guides/sign-in-to-spa-embedded-widget/react/main/#install-the-sdk) and [Load the Sign-in Widget](/docs/guides/sign-in-to-spa-embedded-widget/react/main/#load-the-sign-in-widget) in the [Sign in to your SPA with the embedded Okta Sign-In Widget](/docs/guides/sign-in-to-spa-embedded-widget/react/main/) guide to learn how to set up the Sign-in Widget in your React SPA app.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/widgetforyourapp.md:1]:
```suggestion
See [Install the SDK](/docs/guides/sign-in-to-spa-embedded-widget/react/main/#install-the-sdk) and [Load the Sign-In Widget](/docs/guides/sign-in-to-spa-embedded-widget/react/main/#load-the-sign-in-widget) in the [Sign in to your SPA with the embedded Okta Sign-In Widget](/docs/guides/sign-in-to-spa-embedded-widget/react/main/) guide to learn how to set up the Sign-In Widget in your React SPA app.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/react/applicationtype.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Select **Single-Page Application** as application type, and then click **Next**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/react/applicationtype.md:1]:
```suggestion
Select **Single-Page Application** as the application type, and then click **Next**.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ Run the Okta Identity Engine embedded SDK or embedded Sign-In Widget sample app.
 
 **What you need**
 
-* [Okta org already configured for a password-only use case](/docs/guides/oie-embedded-common-org-setup/-/main/#set-up-your-okta-org-for-a-password-factor-only-use-case)
+* [Okta org already configured for a password-only use case](/docs/guides/oie-embedded-common-org-setup/#set-up-your-okta-org-for-a-password-factor-only-use-case)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:19]:
```suggestion
* [Okta org already configured for a password only use case](/docs/guides/oie-embedded-common-org-setup/#set-up-your-okta-org-for-a-password-factor-only-use-case)
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runsdkapp.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+```shell
+yarn start
+```
+
+6. Open a browser window and navigate to the app's home page: `http://localhost:8080`. Sign in to the sample app with an existing user from your org.
+
+### Troubleshoot
+
+If the app doesn't load in the browser, open the app in private or incognite mode. This allows the app to run with an empty browser cache and may resolve issues related to stale caches.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runsdkapp.md:35]:
```suggestion
If the app doesn't load in the browser, open the app in private or incognito mode. This allows the app to run with an empty browser cache and may resolve issues related to stale caches.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runwidgetapp.md [hunk 8]
```diff
@@ -0,0 +1 @@
+See [Run the sample application](/docs/guides/sign-in-to-spa-embedded-widget/react/main/#run-the-sample-application) in the [Sign in to your SPA with the embedded Okta Sign-In Widget](/docs/guides/sign-in-to-spa-embedded-widget/react/main/) guide to learn how to set up the [React Sign-in Widget sample app](https://github.com/okta/samples-js-react/tree/master/custom-login).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runwidgetapp.md:1]:
```suggestion
See [Run the sample application](/docs/guides/sign-in-to-spa-embedded-widget/react/main/#run-the-sample-application) in the [Sign in to your SPA with the embedded Okta Sign-In Widget](/docs/guides/sign-in-to-spa-embedded-widget/react/main/) guide to learn how to set up the [React Sign-In Widget sample app](https://github.com/okta/samples-js-react/tree/master/custom-login).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/samplecode.md [hunk 9]
```diff
@@ -0,0 +1,2 @@
+* [React sample using the Embedded Sign-in Widget](https://github.com/okta/samples-js-react/tree/master/custom-login)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/samplecode.md:1]:
```suggestion
* [React sample using the Embedded Sign-In Widget](https://github.com/okta/samples-js-react/tree/master/custom-login)
```

**Review by susanharper-okta — APPROVED** [review]
couple nits. approving to not hold you up.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/configlocations.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+    "scopes": [
+        "openid",
+        "profile",
+        ...
+    ],
+    "pkce": true
+ };
+```
+
+And pass this object to  `OktaAuth ` during initialization.
```
[the remark this answers is not in the document]
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/configlocations.md:17]: updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/githubinstructions.md (hunk 1 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/githubinstructions.md:12]: updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md (hunk 2 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md:20]: updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md (hunk 3 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/sdkforyourapp.md:32]: Updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/widgetforyourapp.md (hunk 4 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/react/widgetforyourapp.md:1]: Updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/react/applicationtype.md (hunk 5 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/react/applicationtype.md:1]: updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md (hunk 6 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:19]: updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runsdkapp.md (hunk 7 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runsdkapp.md:35]: updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runwidgetapp.md (hunk 8 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/runwidgetapp.md:1]: Updated.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/samplecode.md (hunk 9 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/react/samplecode.md:1]: Updated.

**Pull request merged**